I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 136, in script
    show koran ypos 0.2
Exception: Image 'koran' does not accept attributes 'ypos 0 .2'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 136, in script
    show koran ypos 0.2
  File "E:\Renpy\renpy-8.3.2-sdk\renpy\ast.py", line 1111, in execute
    show_imspec(self.imspec, atl=getattr(self, "atl", None))
  File "E:\Renpy\renpy-8.3.2-sdk\renpy\ast.py", line 1074, in show_imspec
    renpy.config.show(name,
  File "E:\Renpy\renpy-8.3.2-sdk\renpy\exports\displayexports.py", line 477, in show
    if not base.find_target() and renpy.config.missing_show:
  File "E:\Renpy\renpy-8.3.2-sdk\renpy\display\image.py", line 421, in find_target
    self.target = target._duplicate(a)
  File "E:\Renpy\renpy-8.3.2-sdk\renpy\display\transform.py", line 1188, in _duplicate
    args.extraneous()
  File "E:\Renpy\renpy-8.3.2-sdk\renpy\display\displayable.py", line 109, in extraneous
    raise Exception("Image '{}' does not accept attributes '{}'.".format(
Exception: Image 'koran' does not accept attributes 'ypos 0 .2'.

Windows-10-10.0.22631 AMD64
Ren'Py 8.3.2.24090902
Taksopolis 1.0
Sun Dec  1 22:50:05 2024
